public/app/features/dashboard/components/SubMenu/DashboardLinks.tsx

```diff
@@ -46,7 +46,7 @@ export const DashboardLinks: FC<Props> = ({ dashboard, links }) => {
           <a
             className="gf-form-label gf-form-label--dashlink"
             href={sanitizeUrl(linkInfo.href)}
-            target={link.targetBlank ? '_blank' : '_self'}
+            target={link.targetBlank ? '_blank' : undefined}
```

Comment:
Not sure why this works, isn't _self the default?

Reply:
no, and angular router treats "_self" with some special logic that triggers a full page reload instead of history change